gcp/data_source_gcp_volume.go

```diff
@@ -182,7 +184,7 @@ func dataSourceGCPVolume() *schema.Resource {
 										Computed: true,
 									},
 									"has_root_access": {
-										Type:     schema.TypeBool,
+										Type:     schema.TypeString,
```

Comment:
why not bool?

Reply:
When I look in swagger, I don't see this parameter:
```
ExportPolicy{
description:	
Export Policy for a volume

rules*	[
maxItems: 5
SimpleExportPolicyRule{Jump to definition
description:	
A Simple Export Policy Rule that defines who can access the volume and how. Only 5 rules are allowed but each rule can serve many clients and subnets.

access*	[...]
allowedClients*	[...]
nfsv3	{...}
nfsv4	{...}

Reply:
Are you using the latest swagger?

```bash
curl https://cloudvolumesgcp-api.netapp.com/swagger.json | jq .definitions.SimpleExportPolicyRule_v2.properties.hasRootAccess
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  326k    0  326k    0     0   408k      0 --:--:-- --:--:-- --:--:--  413k
{
  "description": "If enabled (true or on) the rule defines that no_root_squash is set, else if it is disabled (false or off) root_squash is set and user ID mapped to anonymous user 65534. The response will always be true or false and never on/off. If the value is set to null the response will always be set to the default of true.",
  "type": "string",
  "default": "true",
  "enum": [
    "true",
    "false",
    "on",
    "off"
  ],
  "x-nullable": true,
  "example": "true"
}
```

Reply:
Apparently not, though I'm importing the same link in https://editor.swagger.io/.   Maybe it's cached.  it shows 1.1.6

Anyway, this answers my question about string vs bool.